tests/cross/src/test/scala/tests/pc/CompletionOverrideSuite.scala

```diff
@@ -119,7 +132,7 @@ class CompletionOverrideSuite extends BaseCompletionSuite {
   )
 
   checkEdit(
-    "import",
+    "import".tag(IgnoreScala3), // import position is flaky
```

Comment:
I'm not sure what's going on, but I realized that the order of

```
import java.nio.file.FileVisitResult
import java.nio.file.attribute.BasicFileAttributes
import java.nio.file.Path
```

sometimes change in my local environment 🤔